roles/kubernetes/master/templates/manifests/kube-apiserver.manifest.j2

```diff
@@ -100,6 +100,16 @@ spec:
 {% if kube_feature_gates %}
     - --feature-gates={{ kube_feature_gates|join(',') }}
 {% endif %}
+{% if kube_version | version_compare('1.9', '>=') %}
+    - --requestheader-client-ca-file=/etc/kubernetes/ssl/ca.pem
```

Comment:
This should be:
`    - --requestheader-client-ca-file={{ kube_cert_dir }}/ca.pem`